skia/RNKeyEvent.h

```diff
@@ -0,0 +1,41 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+#pragma once
+#include <cxxreact/CxxModule.h>
+#include "ReactSkia/core_modules/RSkInputEventManager.h"
+#include "ReactSkia/utils/RnsUtils.h"
+
+using namespace std;
+using namespace folly;
+
+namespace facebook {
+namespace xplat {
+using namespace std;
+class RNKeyEventModule : public module::CxxModule{
+ private:
+  int  keyEventId_= 0;
+  int callbackId_ = 0;
```

Comment:
Like for "keyEventId_" : 0 is the invalid case, same for "callbackId_ " ?? callbackId_  can be "0" as well